core/domain/classifier_domain.py

```diff
@@ -181,6 +182,8 @@ class ClassifierTrainingJob(object):
             for which the classifier will be generated.
         exp_version: str. The version of the exploration when
             the training job was generated.
+        next_scheduled_check_time: Datetime. The scheduled time to check the
```

Comment:
The type here should be datetime.datetime, I think. https://docs.python.org/2/library/datetime.html#available-types

"The next scheduled time ..."